pkg/networks/usernet/client.go

```diff
@@ -74,11 +75,14 @@ func (c *Client) ResolveAndForwardSSH(ipAddr string, sshPort int) error {
 }
 
 func (c *Client) ResolveIPAddress(ctx context.Context, vmMacAddr string) (string, error) {
-	timeout := time.After(2 * time.Minute)
+	timeout := 2 * time.Minute
+	if os.Getenv("GITHUB_ACTIONS") != "" {
+		timeout = 3 * time.Minute
+	}
 	ticker := time.NewTicker(500 * time.Millisecond)
 	for {
 		select {
-		case <-timeout:
+		case <-time.After(timeout):
```

Comment:
Is this correct? Doesn't it create a new channel each time through the for loop? And since the ticker triggers every 500ms, this would effectively disable the timeout?

Or am I confused about how this would work?

Reply:
Sorry, you are right. 👍